src/main/java/com/derongan/minecraft/deeperworld/config/DeeperConfig.kt

```diff
@@ -20,7 +20,11 @@ object DeeperConfig : IdofrontConfig<DeeperConfig.Data>(deeperWorld, Data.serial
         val remountPacketDelay: TimeSpan = 40.ticks,
         val fall: FallDamageConfig = FallDamageConfig(),
         val time: TimeConfig = TimeConfig(),
-    )
+    ) {
+        val worlds = sections.map {
```

Comment:
I'd prefer this to be a one-liner. Alternatively map on one line and toSet on the other.